src/containers/StudyEdit/style.ts

```diff
@@ -0,0 +1,44 @@
+import styled from '@emotion/styled';
+
+export const StudyEditWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 3rem 1rem;
+  gap: 2rem;
+`;
+
+export const StudyEditHeading = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+`;
+
+export const StudyEditImageWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+`;
```

Comment:
`ImageWrapper` 의 가운데 정렬을 위한 코드입니다!

```suggestion
export const StudyEditImageWrapper = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 1rem;
`;
```

Reply:
이렇게 하니까 Typography도 가운데 정렬이 되어서 ImageWrapper를 ImageContainer로 한번 감싸고, 그 안에서 가운데 정렬하였습니다! 27790bf6f41c5f10a07a56fe0013529d292e9caf